src/main/resources/db/migration/0300/V302__DeliverableInternalInterestMapping.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE accelerator.deliverable_categories
+    ADD COLUMN internal_interest_id INTEGER REFERENCES accelerator.internal_interests ON DELETE SET NULL;
+
+UPDATE accelerator.deliverable_categories
+SET internal_interest_id = id;
```

Comment:
Would it make sense to make this non-nullable after initializing the values, and remove the `ON DELETE SET NULL` from the constratint? Or do we think there'll be deliverable categories that don't map to interests?